_episodes/01-software-development-intro.md

```diff
@@ -50,60 +47,93 @@ smaller sub-modules. Another examples of modules are classes in object-oriented
-interconnected modules:
+MVC architecture divides the related program 
+logic into three interconnected modules:
 
 - **Model** (data)
 - **View** (client interface),  and 
 - **Controller** (processes that handle input/output and manipulate the data).
 
-**Model** represents the data used by a program and also contains operations that act to change the data in the model. 
-This may be a database, a file, or an object - for example a table representing patients data. 
-
-**View** is the means of displaying data to users/clients within an application. For example, displaying a window with input fields and buttons (Graphical User Interface, GUI) or text within a command line shell (Command Line Interface, CLI) are examples of Views. They include anything that the user can see from the application. While building GUIs is not the topic of this workshop, we will cover building CLIs in Python in later episodes. 
-
-**Controller** manipulates both the Model and the View. It accepts input from the View and performs the corresponding action on the Model and then updates the View accordingly. For example, on user request, Controller updates a picture on a user's GitHub profile and then modifies the View by displaying the updated profile back to the user.
+**Model** represents the data used by a program and also contains operations/rules for manipulating and changing the data 
+in the model. 
+This may be a database, a file, a single data object or a series of objects - for example a table representing 
+patients' data. 
+
+**View** is the means of displaying data to users/clients within an application (i.e. provides visualisation of the 
+state of the model). 
+For example, displaying a window with input fields and buttons (Graphical User Interface, GUI) or textual options 
+within a command line shell (Command Line Interface, CLI) are examples of Views. 
+They include anything that the user can see from the application. While building GUIs is not the topic of this workshop, 
+we will cover building CLIs in Python in later episodes. 
+
+**Controller** manipulates both the Model and the View. It accepts input from the View and performs the corresponding 
+action on the Model (changing the state of the model) and then updates the View accordingly. For example, on user 
+request, Controller updates a picture on a user's GitHub profile and then modifies the View by displaying the 
+updated profile back to the user.
       
 Let's have a look at some MVC examples. 
        
-> ## MVC application examples
-> What MVC application examples do you know, either computational or in real life? This exercise can be done as 
+> ## MVC Application Examples
+> What MVC application examples do you know, either from scientific or general applications or real life? This exercise can be done as 
 >a group discussion.
-> > ## Solution
-> > #### Modern web and mobile applications
+> > ## Solution 
```

Comment:
Although certainly relevant, in its current form I'm not sure the blog post style sits well within the exercise solution, since the exercise asks for examples but not the inner workings of them as depicted in the solution. Perhaps remove much of the detail, with the three examples as basic bullet points and without the diagrams?